src/Compilers/Test/Resources/Core/Microsoft.CodeAnalysis.Compiler.Test.Resources.csproj

```diff
@@ -85,11 +86,16 @@
     <Content Include="NetFX\aacorlib\Key.snk" />
     <EmbeddedResource Include="NetFX\Minimal\minasync.dll" />
     <EmbeddedResource Include="NetFX\Minimal\mincorlib.dll" />
+    <EmbeddedResource Include="NetFX\PortableProfile7\mscorlib.dll" />
+    <EmbeddedResource Include="NetFX\PortableProfile7\System.Runtime.dll" />
+    <EmbeddedResource Include="NetFX\WinRt\System.Runtime.WindowsRuntime.dll" />
+    <EmbeddedResource Include="NetFX\WinRt\System.Runtime.WindowsRuntime.UI.Xaml.dll" />
```

Comment:
These two references are available as NuPkg and in theory could be added to the extra sections of Basic.Reference.Assemblies.Net461. Unfortunately, there is a substantial difference between how idiomatic `net461` WinRt projects build using these references and how our tests build. The tests do not use the standard reference versions of these DLLs and using the standard references causes the tests to break. Decided to keep the existing DLLs to maintain test behavior.